CITATION.cff

```diff
@@ -404,5 +404,5 @@ license: "Apache-2.0"
 message: "If you use this software, please cite it using these metadata."
 repository-code: "https://github.com/ESMValGroup/ESMValTool/"
 title: ESMValTool
-version: "v2.10.0"
+version: "v2.11.0"
```

Comment:
Don't forget to update the `date-released` too

Reply:
Done in 54d8a1822; I have added tomorrow's date so I don't need to rush this afternoon 🤪